src/Endpoints/OAuth2.php

```diff
@@ -16,11 +16,8 @@ class OAuth2 extends Endpoint
     /**
      * Get OAuth 2.0 Access Token.
      *
-     * @param string $clientId
      *   The Client ID of your app.
```

Comment:
We keep these lines but remove the `@param` lines? Is that intentional?

Reply:
No, I missed this one. Will update.